### PR TITLE
Add contrib

### PR DESCRIPTION
**Pull request by jean-philippe-martin**
Creating a cozy place for new contributions.

### CODE REVIEW

File: gcloud-java-contrib/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+Google Cloud Java Contributions
+===============================
+
+Additional functionality for gcloud-java (Java idiomatic client for [Google Cloud Platform][cloud-platform] services).
+
+Quickstart
```
Comment (ajkannan) [inline gcloud-java-contrib/README.md]: This section probably should be commented out until we push a release to Maven.  (If a user tried the following, then they would get an error until we do a release).

File: gcloud-java-contrib/pom.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.gcloud</groupId>
+  <artifactId>gcloud-java-contrib</artifactId>
+  <packaging>jar</packaging>
+  <name>GCloud Java contrib</name>
+  <description>
+    gcloud-java additional functionality.
```
Comment (ajkannan) [inline gcloud-java-contrib/pom.xml]: Nit: I think it'd be clearer if we said something along the lines of: "Contains packages that provide higher-level abstraction/functionality for common `gcloud-java` use cases."  If we do change the description here to something more informative, we could use the same description in the README tag line.

File: gcloud-java-contrib/pom.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.gcloud</groupId>
+  <artifactId>gcloud-java-contrib</artifactId>
+  <packaging>jar</packaging>
+  <name>GCloud Java contrib</name>
```
Comment (ajkannan) [inline gcloud-java-contrib/pom.xml]: Nit: could write out "contributions" here just to be consistent with other modules' pom.xmls

**Comment by ajkannan** [comment]
A few minor comments from me, but otherwise looks good!

File: pom.xml [hunk 4]
```diff
@@ -73,6 +73,7 @@
     <module>gcloud-java-examples</module>
     <module>gcloud-java-resourcemanager</module>
     <module>gcloud-java-storage</module>
+    <module>gcloud-java-contrib</module>
```
Comment (mziccard) [inline pom.xml]: Nit: Can you move this so that the list of modules is in alphabetic order?

File: gcloud-java-contrib/README.md (hunk 1 above)
Comment (jean-philippe-martin) [inline gcloud-java-contrib/README.md]: Done.

File: gcloud-java-contrib/pom.xml (hunk 2 above)
Comment (jean-philippe-martin) [inline gcloud-java-contrib/pom.xml]: Done.

File: gcloud-java-contrib/pom.xml (hunk 3 above)
Comment (jean-philippe-martin) [inline gcloud-java-contrib/pom.xml]: Done.

**Comment by jean-philippe-martin** [comment]
Thanks for taking a look! I'm sending an updated version.

File: pom.xml (hunk 4 above)
Comment (jean-philippe-martin) [inline pom.xml]: Sure thing.

**Comment by mziccard** [comment]
@jean-philippe-martin Before merging do you mind squashing the commits? 
@ajkannan Do you have other comments?

**Comment by jean-philippe-martin** [comment]
Squashed; merging when I get the "LGTM".

**Comment by ajkannan** [comment]
LGTM

**Pull request merged**